packages/desktop-client/src/components/reports/Overview.js

```diff
@@ -248,7 +249,6 @@ function Overview({ accounts }) {
         <Card
           style={[
             {
-              color: '#a0a0a0',
```

Comment:
Could you update the color to be a bit gray here (like secondary text color or something)? This stands out too much now.